volumina/utility/abc.py

```diff
@@ -0,0 +1,37 @@
+"""
+Wraps stdlib abc module to provide convenience classes for defining QObject interfaces
+"""
+from abc import ABCMeta, ABC, abstractproperty, abstractclassmethod, abstractmethod
+
+from PyQt5.QtCore import QObject
+
+__all__ = [
```

Comment:
Just for convenience, otherwise compare
```python
from volumina import abc
class MyABC(abc.QABC):
    sig = abc.abstractsignal(int)
    @abc.abstractmethod
    def foo(self): ...
```
to
```python
from volumina.abc import QABC
from abc import abstractmethod
class MyABC(abc.QABC):
    @abstractmethod
    def foo(self): ...
```

Reply:
However, I would still avoid re-exporting `ABC` and `ABCMeta` because they are not required for implementing `QABC`.

Reply:
Valid point. Renamed `utility.abc` to `utility.qabc` and removed export of ABC and ABCMeta